packages/pwa/app/components/_app/index.jsx

```diff
@@ -297,8 +297,7 @@ App.getProps = async ({api}) => {
     const rootCategory = await api.shopperProducts.getCategory({
         parameters: {
             id: DEFAULT_ROOT_CATEGORY,
-            levels: DEFAULT_NAV_DEPTH,
-            locale: localeConfig.app.targetLocale
```

Comment:
This was why `parameters` already contain a locale.

Removing this line and letting our Commerce API client to auto-inject the locale and currency.